src/js/login/containers/Main.jsx

```diff
@@ -128,7 +128,7 @@ class Main extends React.Component {
   }
 
   handleLogin(loginUrl = 'idme') {
-    this.loginUrlRequest = handleLogin(this.props.login.loginUrls[loginUrl], this.props.onUpdateLoginUrl);
+    this.loginUrlRequest = handleLogin(this.props.login.loginUrls[loginUrl], this.props.updateLogInUrls);
```

Comment:
Hm, looking this over, I'm not sure this second argument really serves any purpose... `handleLogin`, if given the `loginUrl`, ends up doing `getLoginUrls`, but presumably we already have that in order to pass it into the function.